e2e/cypress/integration/notifications/desktop_notifications_spec.js

```diff
@@ -105,3 +179,29 @@ const stubNotificationAs = (name, permission) => {
         cy.stub(win, 'Notification').as(name);
     });
 };
+
+const changeDesktopNotificationSettingsAs = (category) => {
+    // # Click hamburger main menu.
+    cy.get('#sidebarHeaderDropdownButton').click();
+
+    // # Click "Account settings"
+    cy.findByText('Account Settings').should('be.visible').click();
+
+    // * Check that the "Account Settings" modal was opened.
+    cy.get('#accountSettingsModal').should('exist').within(() => {
+        // # Click "Notifications"
+        cy.findByText('Notifications').should('be.visible').click();
+
+        // # Click "Desktop"
+        cy.findByText('Desktop Notifications').should('be.visible').click();
+
+        // # Select category.
+        cy.get(category).check();
+
+        // # Click "Save"
+        cy.findByText('Save').should('be.visible').click();
```

Comment:
I'm getting failing test here since "Save" button is covered. I suggests,
```suggestion
        cy.findByText('Save').scrollIntoView().should('be.visible').click();
```